include/unifex/sender_concepts.hpp

```diff
@@ -285,19 +296,19 @@ using operation_t [[deprecated("Use connect_result_t instead of operation_t")]]
 template <typename Sender, typename Receiver>
 [[deprecated("Use sender_to instead of is_connectable_v")]]
 inline constexpr bool is_connectable_v =
-  is_callable_v<tag_t<connect>, Sender, Receiver>;
+  is_callable_v<decltype(connect), Sender, Receiver>;
```

Comment:
I wonder why we made this change.

Reply:
`main` was updated to use `tag_t`, I'll keep that